.storybook/preview-head.html

```diff
@@ -4,16 +4,31 @@
+    @font-Face{font-family:"Guardian Text Egyptian Web"; src:url("./public/fonts/GuardianTextEgyptian-Reg.ttf");}
+    @font-Face{font-family:"Guardian Text Egyptian Web"; font-style:"italic"; src:url("./public/fonts/GuardianTextEgyptian-RegItalic.ttf")}
+    @font-Face{font-family:"Guardian Text Egyptian Web"; font-weight:500; src:url("./public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    @font-Face{font-family:"Guardian Text Egyptian Web"; font-weight:"bold"; src:url("./public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    @font-Face{font-family:"Guardian Text Egyptian Web"; font-weight:"bold"; font-style:"italic"; src:url("./public/fonts/GuardianTextEgyptian-BoldItalic.ttf")}
+
+    @font-Face{font-family:"Guardian Text Sans Web"; font-weight:300; src:url("./public/fonts/GuardianTextSans-Regular.ttf")}
+    @font-Face{font-family:"Guardian Text Sans Web"; font-weight:300; font-weight:"italic"; src:url(".public/fonts/GuardianTextSans-RegularItalic.ttf")}
+    @font-Face{font-family:"Guardian Text Sans Web"; font-weight:600; src:url("./public/fonts/GuardianTextSans-Bold.ttf")}
+    @font-Face{font-family:"Guardian Text Sans Web"; font-weight:600; font-weight:"italic"; src:url(".public/fonts/GuardianTextSans-BoldItalic.ttf")}
+    
+    @font-Face{font-family:"Guardian Headline"; font-weight:200; src:url("/public/fonts/GHGuardianHeadline-Light.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:200; font-weight: "italic"; src:url("./public/fonts/GHGuardianHeadline-LightItalic.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:400; src:url("/public/fonts/GHGuardianHeadline-Regular.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:400; font-weight: "italic"; src:url("./public/fonts/GHGuardianHeadline-RegularItalic.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:500; src:url("/public/fonts/GHGuardianHeadline-Medium.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:500; font-weight: "italic"; src:url( "./public/fonts/GHGuardianHeadline-MediumItalic.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:600; src:url("/public/fonts/GHGuardianHeadline-Semibold.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:600; font-weight: "italic"; src:url( "./public/fonts/GHGuardianHeadline-SemiboldItalic.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:700; src:url("/public/fonts/GHGuardianHeadline-Bold.ttf")}
+    @font-Face{font-family:"Guardian Headline"; font-weight:700; font-weight: "italic"; src:url("./public/fonts/GHGuardianHeadline-BoldItalic.ttf")}
+
+    @font-Face{font-family:"Guardian Icons"; src:url("/public/fonts/icons.otf")}
+
+    body{margin:0;font-family:'Guardian Text Egyptian Web';font-size:1.6em;line-height:1.5em;overflow-x:hidden;}
     figure{margin:1em 0;}
     figure.element-embed{margin:2em 0;}
     .js-email-sub__iframe{width:100%;}
```

Comment:
As @paperboyo mentioned offline, could the 500 weight fonts be referred to as "medium", and the 700 weights be referred to as "bold"? Would that cause any problems?

Also, a number of the `font-face` declarations in this file refer to `font-weight:"italic"`... do you perhaps mean `font-style:"italic"`?

Reply:
I’m as familiar with CSS as American Dad fighting windows blinds, but my understanding (or the lack of it) is:
1. Correct numbers for weights are like that (forget the font name, it applies to weights from all of them):
![image](https://user-images.githubusercontent.com/6032869/72005952-a4128c80-3246-11ea-9447-02011939f637.png)
So they should be in the code *unless* some (old?) content may ask for `500` meaning Bold, but since bold will be `700`, the call we actually end up returning Regular. If that’s the case, we may want to decide to keep wrong numbers for the sake of bakwards compatibility.
2. `font-weight:"italic"` indeed sounds bonkers 😆